_includes/templates/install/ubuntu-queue-pub-sub.md

```diff
@@ -0,0 +1,42 @@
+#### Google Pub/Sub Configuration
+
+To access Pub/Sub service, you first need to create an [Google cloud account](https://cloud.google.com/).
+
+To work with Pub/Sub service you will need to create a project using [this instruction](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+Create service account credentials using [this instruction](https://cloud.google.com/pubsub/docs/quickstart-py-mac#create_service_account_credentials),
+and save json file with your service account credentials step 9. [here](https://cloud.google.com/pubsub/docs/quickstart-py-mac#create_service_account_credentials).
+
+You will need to change following Queue type parameter in [thingsboard.yml](/docs/user-guide/install/config/#thingsboardyml):
+
+```bash
+queue:
+  type: "${TB_QUEUE_TYPE:pubsub}"
+```
+
+Add your credentials in [thingsboard.yml](/docs/user-guide/install/config/#thingsboardyml).  Don’t forget to replace “YOUR_PROJECT_ID”, "YOUR_SERVICE_ACCOUNT" with your **real Pub/Sub project id:,
+ and service account from saved json file.**
```

Comment:
please specify that YOUR_SERVICE_ACCOUNT is the whole json file

Reply:
done